python_modules/libraries/dagster-elt/dagster_elt/asset_defs.py

```diff
@@ -0,0 +1,53 @@
+from typing import List, Iterable, Optional, Union
+from dagster import AssetsDefinition, asset, AssetKey, AssetExecutionContext
+from dagster._core.definitions.events import CoercibleToAssetKey
+from . import SlingMode, SlingResource, SlingSource, SlingTarget
+import re
+
+
+def build_sling_asset(
+    key: AssetKey,
+    sling_resource_key: str,
+    source_table: str,
+    dest_table: str,
+    mode: object = SlingMode.FULL_REFRESH,
+    primary_key: Optional[Union[str, List[str]]] = None,
+    update_key: Optional[Union[str, List[str]]] = None,
+    deps: Optional[Iterable[CoercibleToAssetKey]] = None,
+) -> AssetsDefinition:
+    """Factory which builds an asset definition that syncs the given source table to the given
+    destination table.
+    """
```

Comment:
I think including args-level docs is worthwhile for a function like this.

Reply:
For sure, will include once I've nailed the API